Fix ciflow/all label generation

zhouzhuojie opened the pull request:
the `ciflow/all` is automatically added but need to be added before we call `gen_root_job_condition`.

- fix the order of adding `ciflow/all`
- refactor all the string into global constants

malfet commented on .github/scripts/generate_ci_workflows.py:
```diff
@@ -73,6 +73,9 @@ def __post_init__(self) -> None:
             return
+
+        # Add ciflow/all to labels
+        self.labels.add('ciflow/all')
```
My favorite remark: please replace string global variable (so that linter can catch typos in those)
```suggestion
        self.labels.add(CIFLOW_ALL)
```

zhouzhuojie replied:
ok, for consistency I will change all of these into global constants, changes coming up

zhouzhuojie committed:
Fix ciflow/all label generation (#63954)

Summary:
the `ciflow/all` is automatically added but need to be added before we call `gen_root_job_condition`.

- fix the order of adding `ciflow/all`
- refactor all the string into global constants

Pull Request resolved: https://github.com/pytorch/pytorch/pull/63954

Differential Revision: D30545596

Pulled By: zhouzhuojie

fbshipit-source-id: c5258bd454491db2b418b003f97d7c16679ae9d2
